src/components/CallView/shared/VideoVue.vue

```diff
@@ -475,6 +490,14 @@ export default {
 
 		// Set initial state
 		this._setStream(this.model.attributes.stream)
+
+		if (this.isBig) {
+			this.$refs.video.addEventListener('playing', this.getVideoAspectRatio)
+		}
+	},
+
+	beforeDestroy() {
+		this.$refs.video.removeEventListener('playing', this.getVideoAspectRatio)
 	},
```

Comment:
In Vue component's elements events are usually set with `v-on` directive in the template. Then all component's DOM events would be defined in the template and we wouldn't need to remove listeners manually.

```html
<video @playing="updateVideoAspectRatio"></video>
```

By the way, do we need to handle this event to update the aspect ratio? I'm not sure, but it seems to me that anytime the video is changed the stream would be also changed. Then the `model.attributes.stream` watcher with `_setStream` handler will be triggered. So we may calculate the aspect ratio immediately on the stream receiving from `model.attributes.stream.getVideoTracks()[0].getSettings().aspectRatio` while attaching the media stream before mounting to the DOM.

Reply:
Unfortunately, this method doesn't return aspectRatio neither after setting stream, nor after `nextTick` call (see below):

![image](https://user-images.githubusercontent.com/93392545/217023495-99aa4bee-38b7-4fac-a39b-57bacf8d627e.png)


these params appear in the settings only when video is on, which is - the best i could find - right after `playing` event
```
{
    "aspectRatio": 1.3333333333333333,
    "frameRate": 30,
    "height": 360,
    "width": 480
}
```